addons/skin.estuary/xml/DialogPVRInfo.xml

```diff
@@ -1,126 +1,372 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <window>
-	<defaultcontrol always="true">9000</defaultcontrol>
+	<defaultcontrol always="true">5000</defaultcontrol>
 	<onload>SetProperty(infobackground,$ESCINFO[ListItem.Art(fanart)],home)</onload>
 	<onunload>ClearProperty(infobackground,home)</onunload>
 	<controls>
-		<control type="group">
-			<animation effect="fade" start="0" end="100" time="100">WindowOpen</animation>
-			<animation effect="fade" start="100" end="0" time="100">WindowClose</animation>
-			<animation effect="fade" start="100" end="75" time="0" condition="true">Conditional</animation>
-			<visible>Window.isActive(fullscreenvideo)</visible>
-			<include>ColoredBackgroundImages</include>
-		</control>
 		<control type="group">
 			<centertop>50%</centertop>
 			<height>1080</height>
 			<centerleft>50%</centerleft>
 			<width>1920</width>
-			<control type="image">
+			<control type="group">
+				<top>150</top>
+				<left>66</left>
 				<include>OpenClose_Left</include>
-				<left>114</left>
-				<top>236</top>
-				<width>461</width>
-				<height>461</height>
-				<aspectratio>keep</aspectratio>
-				<texture fallback="DefaultTVShows.png">$INFO[ListItem.Icon]</texture>
+				<control type="group">
+					<control type="image">
```

Comment:
```suggestion
					<control type="image">
						<top>-16</top>
						<left>-16</left>
						<width>566</width>
						<height>718</height>
						<aspectratio>stretch</aspectratio>
						<texture border="21">dialogs/dialog-bg.png</texture>
					</control>
```
Image control properties not indented.

Reply:
Don't think I used the Github Suggestion feature right as I couldn't see how to do a code block rather than a single line.

Reply:
No problem. I will manually apply the change, then force-push.